src/LfMergeBridge/LanguageForgeCommitAuxiliaryActionHandler.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Copyright (c) 2016 SIL International
+// This software is licensed under the MIT License (http://opensource.org/licenses/MIT)
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.IO;
+using System.Reflection;
+using Chorus.sync;
+using Chorus.VcsDrivers;
+using LibFLExBridgeChorusPlugin.Infrastructure;
+using LibTriboroughBridgeChorusPlugin;
+using LibTriboroughBridgeChorusPlugin.Infrastructure;
+using SIL.Code;
+using SIL.Progress;
+
+namespace LfMergeBridge
+{
+	/// <summary>
+	/// Auxiliary action handler. This is action handler is used in a test tool for LfMerge.
+	/// It does the local, i.e. commit, part of a sync action and helps to re-create the data
+	/// used in LfMerge's integration tests after updating or modifying the FW data used in those
+	/// tests.
+	/// </summary>
+	[Export(typeof(IBridgeActionTypeHandler))]
+	internal class LanguageForgeCommitAuxiliaryActionHandler: IBridgeActionTypeHandler
```

Comment:
So this looks to be included in production even though it is for testing. Is it for tracking down problems on live data?

Reply:
Pretty much yes.